### DISCUSSION TITLE
Everything woprks but logs constantly show errors

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#3410] [Ask A Question]
Everything seems to work fine, but the logs constantly show errors like this:

![image](https://user-images.githubusercontent.com/118814/176955797-f5859ed2-b0be-42ab-9115-72d735080241.png)

Is this normal or something to resolve?

**Comment by NickM-27** [comment]
The `Non-monotonous DTS in output...` is an error with the incoming data, usually caused by WiFi camera and some interference but can happen for other reasons too. It doesn't take the stream down and there likely isn't much that can be done.

The `Too many bites > clamping to max` is based on the fact that the audio the camera just sent is being clamped down to the size that ffmpeg detected when starting the stream. You could update your ouptut args to set a higher `ar` but it is also relatively harmless assuming the audio in your recordings is still working as expected.

Anything with `terminating ffmpeg process` is a hard error and caused the stream to restart, could again be from bad connection / some corrupt data from camera or something else. As long as this doesn't happen often I wouldn't worry.
